docs/templates/index.html

```diff
@@ -55,8 +55,8 @@ <h2>Get started quickly</h2>
         <a href="/docs/quickstart" class="btn btn-secondary">Quickstart »</a>
       </div>
       <div class="col-12 col-lg-6 mt-4 mt-lg-0">
-        <pre><code class="language-sh">pip install dash-bootstrap-components</code></pre>
-        <pre><code class="language-python">import dash
+        <pre><code class="language-sh border">pip install dash-bootstrap-components</code></pre>
+        <pre><code class="language-python border">import dash
```

Comment:
See screenshot in main comment, there's some padding / margin issue here I think

Reply:
I see the issue in the image you posted, but I can't duplicate it.  I tried Chrome, Firefox and  various screen sizes, but it looks fine to me.  How do I duplicate what you are seeing?

![image](https://github.com/facultyai/dash-bootstrap-components/assets/72614349/cc9285ae-5783-43ef-bc00-788283bd9ee8)

Reply:
That's really weird... it's the `margin-bottom` property on `<pre>` elements that's causing it. We need to set that to zero.

Can you see what the margin is on `<pre>` elements by checking in the inspector?

I'd be happy to merge this and tweak with a follow-up though.